workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/matplotlib.py

```diff
@@ -24,23 +24,29 @@
 template = jinja2.Template(
     """
 <h2> {{varfilter}} </h2>
-<table cellpadding="0" cellspacing="0">
+<table class="display" style="width:100%">
 
+<thead>
 <tr>
+<th>Variable</th>
```

Comment:
Noah had a slack comment about making this bigger

Reply:
Increased size by 15%